src/utils/constants.ts

```diff
@@ -13,6 +13,6 @@ export const EXPECTED_PERMISSIONS_BLOCK = [
 // The version of the environment in which a replay is executed. This should be bumped whenever
 // the environment changes in a way that would cause a replay to behave differently, e.g. upgrading to a newer
 // replay-orchestrator-launcher version, or changing the version of puppeteer.
-export const LOGICAL_ENVIRONMENT_VERSION = 2;
+export const LOGICAL_ENVIRONMENT_VERSION = 3;
```

Comment:
Version bump due to `moveBeforeClick` being toggled to true as part of the default options switch.